src/Diagrams/Backend/Postscript.hs

```diff
@@ -40,14 +42,17 @@ module Diagrams.Backend.Postscript
   , B
 
     -- * Postscript-specific options
+    -- | #postscript-options#
     -- $PostscriptOptions
 
   , Options(..), psfileName, psSizeSpec, psOutputFormat
 
     -- * Postscript-supported output formats
+
   , OutputFormat(..)
 
-  , renderDias
```

Comment:
I see no reason to stop exporting `renderDias`; that would just unnecessarily break anyone's code who happens to be using it.

Reply:
Make sense, it is exported again.